docs/rules/no-classic-classes.md

```diff
@@ -38,13 +38,19 @@ import Evented from '@ember/object/evented';
 export default class MyComponent extends Component.extend(Evented) {}
 ```
 
+## Configuration
```

Comment:
It's probably worth adding an invalid example above to demonstrate how to use this option:

```js
// With option: additionalInvalidImports = ['my-custom-addon']

import CustomClass from 'my-custom-addon';

export default CustomClass.extend({});
```

Reply:
Agree, good callout.